### ISSUE TITLE
Should ints always be declared as floats?

### ISSUE THREAD

**Issue by vitreo12** [vitreo12/omni#63]
Should `a = 0` be interpreted as `var a : float = 0` ?

**Comment by vitreo12** [comment]
This creates a lot of confusion, perhaps this is a better solution:

```nim
a = 0.0   #inferred to be float
a = 2      #will be turned into a = float(2)
```

**Comment by vitreo12** [comment]
Would also fix #61

**Comment by vitreo12** [comment]
Fix #65
